ansible/README.md

```diff
@@ -2,32 +2,35 @@
 
 These are a set of reference playbooks to configure an application project namespace on a set of clusters.
 
-1. Edit file `project_bootstrap.sh` and run on appropriate clusters.
+1. Using the 'group_vars/\*.yml.example' files, rename each file to remove '.example'.  Make sure to minimally edit the following:
+  - **all.yml**: `central_registry_hostname`
+  - **[group].yml**: `clusterhost`
+  - **[group].yml**: `*_users`
 
-        ./project_bootstrap.sh oc login DEVURL:PORT --token SESSION_TOKEN
-        ./project_bootstrap.sh oc login STAGEURL:PORT --token SESSION_TOKEN
-        ./project_bootstrap.sh oc login PRODURL:PORT --token SESSION_TOKEN
-        ./project_bootstrap.sh oc login REGISTRYURL:PORT --token SESSION_TOKEN
 
-1. The registry ansible token does not need as much privilege. Let's tweak that:
+        ansible/
+        ├── group_vars
+        │   ├── all.yml
+        │   ├── dev.yml
+        │   ├── prod.yml
+        │   ├── registry.yml
+        │   └── stage.yml
 
-        export PROJECT=lifecycle  # the name of the project in the registry host
-        oc policy remove-role-from-user admin -n $PROJECT -z ansible
-        oc policy add-role-to-user registry-admin -n $PROJECT -z ansible
+2. Using the 'host_vars/[environment]-1.yml.example' files, rename each file to remove '.example'.  Depending on your authentication method to OpenShift either use:
+  - `openshift_username`
+  - `openshift_password`
```

Comment:
Being a bit picky here but... I think this could be a bit more clear/verbose, clarifying "either use username+password OR token", and possibly mention why (e.g. if your auth mechanism requires login to an external auth provider) and how (`oc whoami -t`) you would use token instead of user/pwd.

It could be good to reference anisble-vault too given that your change enables its use...